assets/app/scripts/controllers/settings.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:ServicesController
+ * @description
+ * # ProjectController
+ * Controller of the openshiftConsole
+ */
+angular.module('openshiftConsole')
+  .controller('SettingsController', function ($scope, DataService, $filter, LabelFilter, $timeout) {
+    $scope.quotas = {};
+    $scope.limitRanges = {};
+    $scope.labelSuggestions = {};
+    $scope.alerts = $scope.alerts || {};
+    $scope.emptyMessageQuotas = "Loading...";
+    $scope.emptyMessageLimitRanges = "Loading...";
+    $scope.renderOptions = $scope.renderOptions || {};
+    $scope.renderOptions.hideFilterWidget = true;
+    var watches = [];
+
+    DataService.list("resourceQuotas", $scope, function(quotas) {
+      $scope.quotas = quotas.by("metadata.name");
+      $scope.emptyMessageQuotas = "There are no resource quotas set on this project.";
+      console.log("quotas", $scope.quotas);
+    });
+
+    DataService.list("limitRanges", $scope, function(limitRanges) {
+      $scope.limitRanges = limitRanges.by("metadata.name");
+      $scope.emptyMessageLimitRanges = "There are no resource limits set on this project.";
+      console.log("limitRanges", $scope.limitRanges);
+    });    
+
+    $scope.$watch("quotas", function() {
```

Comment:
Is there a cleaner way to trigger this? Maybe using a directive to render tooltips?